example_packages/profiles_priorities/d1/source/d1_m.F90

```diff
@@ -0,0 +1,13 @@
+module d1_m
+    use d11_m, only: create_greeting
+    use d12_m, only: get_name
+    implicit none
+
+    public :: say_hi
+contains
+    subroutine say_hi()
+        if (not_defined /= 3) stop
```

Comment:
`stop` will always return with exit code 0

Reply:
So should I change it to stop 1?